### PR TITLE
Ie 1093 create policy constrains using terraform vet

### PR DESCRIPTION
**Pull request by rmaloloyon**
**PR CONTENT:**
_Use GCP Google Storage Constraints_
- gcp_storage_bucket_retention_v1
- gcp_storage_location_v1_

### CODE REVIEW

File: policies/templates/gcp_storage_bucket_retention_v1.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@ spec:
         	violations := get_diff
         	count(violations) > 0
         	violation_msg := concat(" & ", violations)
-        	message := sprintf("Storage bucket %v has a retention policy violation: %v", [asset.name, violation_msg])
+        	message := sprintf("Storage bucket has a retention policy violation: %v", [asset.name, violation_msg])
```
Comment (leylmordor) [inline policies/templates/gcp_storage_bucket_retention_v1.yaml]: @rmaloloyon lets not change these values?

Comment (rmaloloyon) [reply policies/templates/gcp_storage_bucket_retention_v1.yaml]: Yes sure @leylmordor, but the expected output could be something like this. Try to revert it to the original constraints
```
 //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs does not have
 'Storage bucket //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs
```
Not sure on how to increase the limit while displaying the output here in GUI.

Comment (rmaloloyon) [reply policies/templates/gcp_storage_bucket_retention_v1.yaml]: I modified it a little bit with the template and I got these results, 
```---------------------------------
Constraints Diagnostic Analysis :
---------------------------------

 Does not have bucket policy only enabled - //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs
 It has a retention policy violation - Storage bucket //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs

Policy: 17 test, 15 passed, 0 warning, 2 failure, 0 exceptions```

File: policies/constraints/bq_dataset_location.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  name: bq_dataset_allowed_locations
+  annotations:
+    description: Checks in which locations BigQuery datasets exist.
+    bundles.validator.forsetisecurity.org/healthcare-baseline-v1: security
+spec:
+  severity: high
+  parameters:
+    mode: "allowlist"
+    locations:
+    - us-west2
```
Comment (leylmordor) [inline policies/constraints/bq_dataset_location.yaml]: This should be `us-central` or `us-east4`

**Review by leylmordor — CHANGES_REQUESTED** [review]
@rmaloloyon lets not change the templates message? as those variables could be of use?

**Review by leylmordor — APPROVED** [review]

**Pull request merged**